foundation/core/alerts.tf

```diff
@@ -0,0 +1,20 @@
+# Code adapted from:
+# https://github.com/dzeyelid/azure-cost-alert-webhook-to-slack/blob/main/iac/terraform/main.tf
+
+module "billing_alert_functions" {
+  providers = {
+    azurerm = azurerm.management
+  }
+  source = "../../modules/azure/billing-alert-functions"
+
+  to_provision       = true # feature flag
```

Comment:
We shouldn't need to pass a feature flag to the module itself.  If the billing alert functions aren't enabled, we can use the `count` meta argument to decide whether or not this module is used.

Reply:
To add to this, after removing this variable, we can remove all of the checks that are used within the module, and any reference to these resources that are using `count.index`.